From: Robert Marko <robimarko@gmail.com>
Date: Fri, 6 Jan 2023 16:33:55 +0000 (+0100)
Subject: devices: add Qualcomm Atheros QCN6024/9024/9074 cards
X-Git-Url: http://git.openwrt.org/?p=project%2Fiwinfo.git;a=commitdiff_plain;h=c7b420a2f33c6f1034c3e2191eba0cb0374af7b6

devices: add Qualcomm Atheros QCN6024/9024/9074 cards

Add Qualcomm Atheros QCN6024/9024/9074 PCI ID, they all are compatible and
use the same ID.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---

--- a/hardware.txt
+++ b/hardware.txt
@@ -173,6 +173,7 @@
 0x168c 0x0046 0x168c 0xcafe    0      0  "Qualcomm Atheros" "QCA9984"
 0x168c 0x0050 0x0000 0x0000    0      0  "Qualcomm Atheros" "QCA9887"
 0x168c 0x0056 0x0000 0x0000    0      0  "Qualcomm Atheros" "QCA9886"
+0x17cb 0x1104 0x17cb 0x1104    0      0  "Qualcomm Atheros" "QCN6024/9024/9074"
 0x1814 0x3050 0x1814 0x0005    0      0  "Ralink"   "Rt3050"
 0x1814 0x3051 0x1814 0x0007    0      0  "Ralink"   "Rt3051"
 0x1814 0x3052 0x1814 0x0008    0      0  "Ralink"   "Rt3052"
